src/components/ebay-skeleton/examples/grouped-tile.marko

```diff
@@ -0,0 +1,32 @@
+<style>
+    .group-demo-tile-container{
+        display: flex;
+    }
+    .demo-tile-container {
+        margin-right: 20px;
+        width: 220px;
+    }
+    .custom-image {
+        height: 220px;
+        width: 220px;
+    }
+</style>
+
+<ebay-skeleton class="group-demo-tile-container">
```

Comment:
I would add a specific `a11yText` to this just to show that its custom

Reply:
I don't think it is necessary, it is consistent with other examples. There is no custom a11y text because of a grouped variant. It would be still loading IMO.